ci/fastai_integration_tests.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -x
+
+# https://stackoverflow.com/questions/42218009/how-to-tell-if-any-command-in-bash-script-failed-non-zero-exit-status/42219754#42219754
+# Set err to 1 if pytest failed.
+error=0
+trap 'error=1' ERR
+
+GIT_ROOT=$(git rev-parse --show-toplevel)
+cd "$GIT_ROOT" || exit
+
+# Install PyTorch
+pip install fastai==2.2.5
```

Comment:
Do we need to pin the version here?

Reply:
I guess not, Fastai 2 is the default now rt?

Reply:
yep